gio/src/task.rs

```diff
@@ -60,6 +60,51 @@ impl Task {
         }
     }
 
+    #[doc(alias = "g_task_run_in_thread")]
+    pub fn run_in_thread<Q>(&self, task_func: Q)
+    where
+        Q: FnOnce(&Self, Option<&glib::Object>, Option<&Cancellable>),
+        Q: Send + 'static,
+    {
+        let task_func_data = Box_::new(task_func);
+
+        // FIXME: we store the func pointer into the task data
+        // for now this is okaish, since we do not expose a way to
+        // set the task data in the bindings
```

Comment:
Where else would you store it?

Reply:
Yeah, I think this is the only place where we can store it: I left the fixme just to double check with you.

Shall I explicitly mark set_data as ignore in the gir.toml()

Reply:
Would make sense, thanks :)